chain/chain/src/chain.rs

```diff
@@ -4336,6 +4348,24 @@ pub struct BlockCatchUpResponse {
     pub results: Vec<Result<ApplyChunkResult, Error>>,
 }
 
+#[derive(Message)]
+#[rtype(result = "()")]
+pub struct StateSplitRequest {
+    pub sync_hash: CryptoHash,
+    pub shard_id: ShardId,
```

Comment:
nit: shard_id can be get from shard_uid from shard_uid.shard_id(), so it's a little redundant to have both here